release-process.md

```diff
@@ -1,54 +1,131 @@
+
+
+- As an **individual engineer**, you're empowered and responsible to start release candidates on your own when you see fit.
+- As an **engineering team**, we're collectively empowered and responsible to reliably release our software to production.
+
+This is accomplished by decentralizing the process itself so that any engineer can bring their [production ready](production-ready.md) and [review approved](code-review.md) code to production.
+
+## Responsibilities
+
+### Individual Engineer
+
+As an engineer you have responsibilities to ensure the work you've done is reliably delivered to production:
+
 
-Our release process prepares an application for
-release to users. Releases are made in the context of a GitHub
-repository. This document describes the steps required.
+- Before your code is merged:
+  - Coordinate with DevOps on having configuration in place for your changes _before_ it gets deployed.
+- After your code is merged:
+  - Start a release candidate for your changes (and any other changes others may have committed) at some point within the next 12-24 hours of merging your changes.
+  - When the changes have been deployed on RC, give your changes a thorough round of testing.
+- After your code is released:
+  - When the release is deployed to production, monitor Sentry for possible errors that might indicate we need to revert.
+  - If you feel a release needs to be reverted, communicate this to the team and have an engineer who has permissions to do so take this action.
+
+### Engineering Team
+
+Collectively the team is responsible for coordinating the release of a set of changes to production:
+
+- The engineer who started the release candidate is responsible for taking the actions to release it to production as well.
+  - If you will not be available for this, communicate with the team in order to have someone else cover you.
+- The team validates the release candidate:
+  - As noted above, individuals validate and check off their boxes on the release candidate.
+  - The team should communiate amongst those working on a project on what changes are included in a release candidate.
+  - If a release candidate needs to be redone because of a bug or regression, that should be communicated so the work can be prioritized.
+- When the release candidate has been validated by the team, the engineer who started the release candidate releases it to production.
+
+The team should weight the following factors to determine if a release is good to move forward with:
+- Has the release candidate been adequately tested on RC?
+- Does production have required configuration in place?
+- In the timing of this release appropriate?
+  - Typically we avoid end-of-day or end-of-week releases because a bug could cause unnecessary off-hours intervention.
```

Comment:
Do we want to formalize it and have "No Change Fridays" and a daily 4PM ET cut-off as well? And them exemptions are as simple as: "Peter says so" and "the site is down"?